src/procedures/procedures.11tydata.yaml

```
@@ -0,0 +1,284 @@
+---
+procedureNewZigbee:
+  title: Setting up a new Zigbee network using SkyConnect
+  steps:
+    - title: Plug the extension cable into SkyConnect
+      image: skyconnect-raspi-01.jpg
+      content: |
+        <ol>
+          <li>Plug the Home Assistant SkyConnect into the USB extension cable.</li>
+          <li>Plug the extension cable into your Home Assistant instance.</li>
+          <li class="warning">It is important to use the USB extension cable. USB 3.0 ports and USB 3.0 drives can cause interference with 2.4 GHz protocols, including the ones provided by Home Assistant SkyConnect.</li>
```

I've found people sometimes use the USB extension cable but still physically place the SkyConnect near the interference source, re-introducing the problem the USB extension cable was meant to fix:

```suggestion
          <li class="warning">USB 3.0 ports and USB 3.0 drives can cause interference with 2.4 GHz protocols, including the ones provided by Home Assistant SkyConnect. It is important to use the USB extension cable and to position the Home Assistant SkyConnect away from possible interference sources.</li>
```